.github/actions/spelling/expect.txt

```diff
@@ -287,6 +287,7 @@ powertoys
 pri
 processthreads
 productcode
+PROGRAMFILES
```

Comment:
nit: this should probably move to allow.txt as this is well known Windows env var

Reply:
Moved this to allow.txt